website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -112,6 +112,65 @@ For Slack and Microsoft Teams notifications, the payload conforms to the respect
 }
 ```
 
+### Workspace Notification Payload
+
+| Name                                         | Type   | Description                                                               |
+| -------------------------------------------- | ------ | ------------------------------------------------------------------------- |
+| `payload_version`                            | number | Currently always "2".                                                     |
```

Comment:
We should avoid time-bound language in the docs if possible :-) 

```suggestion
| `payload_version`                            | number | Always "2".                                                     |
```

Reply:
This is copied from the run payload above.  I will change it there, too.